### PR TITLE
Add section label with role and aria-labelledby

### LINKED ISSUE #2689: Create Most Read Section with role and aria label
**Is your feature request related to a problem? Please describe.**
We should create Most Read Section with role and aria label in the `index` file

Acceptance criteria

Most Read - Landmark - Article

    Given I use a screen reader
    When I navigate to the Most Read section on an article page
    Then a region landmark must be announced e.g. Region
    And this must be announced with a label 
    And the label must be the same as the h2 (e.g. Most read) and be unique to the page
    And this should not be inside the main landmark or any other landmark

Most Read - Landmark - Index page

    Given I use a screen reader
    When I navigate to the Most Read section on an index page
    Then a region landmark must be announced e.g. Region
    And this must be announced with a label 
    And the label must be the same as the h2 (e.g. Most read) and be unique to the page
    And this should be inside the main landmark

**Describe the solution you'd like**
- A Section with role and aria label. The label should link to an id on the h2 `MostRead`

Screenshot of similar implementation on front pages:
Screenshot of front page section

This should 

**Describe alternatives you've considered**

**Testing notes**
we should have snapshot test for the section

**Additional context**

### PR DESCRIPTION
**Pull request by EinsteinNjoroge**
Resolves #2689 

**Overall change:** _Create Most Read Section with role and aria-label._

**Code changes:**

- _Wrap MostRead title inside section with `role` and `aria-label `attributes._
- _update snapshots._

Screen Shot 2019-12-02 at 11 02 38 AM

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by EinsteinNjoroge** [comment]
This approach does not use [`psammead-section-label`](https://github.com/bbc/psammead/tree/latest/packages/components/psammead-section-label) since that would introduce a lot of unnecessary css.

File: packages/components/psammead-most-read/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 0.2.1-alpha.0 | [PR#2719](https://github.com/bbc/psammead/pull/2719) Add section label with role and aria-labelledby |
```
Comment (DenisHdz) [inline packages/components/psammead-most-read/CHANGELOG.md]:
```suggestion
| 0.2.1-alpha.1 | [PR#2719](https://github.com/bbc/psammead/pull/2719) Add section label with role and aria-labelledby |
```

File: packages/components/psammead-most-read/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-most-read",
-  "version": "0.2.0-alpha.2",
+  "version": "0.2.1-alpha.0",
```
Comment (DenisHdz) [inline packages/components/psammead-most-read/package.json]:
```suggestion
  "version": "0.2.1-alpha.1",
```

File: packages/components/psammead-most-read/package-lock.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-most-read",
-  "version": "0.2.0-alpha.2",
+  "version": "0.2.1-alpha.0",
```
Comment (DenisHdz) [inline packages/components/psammead-most-read/package-lock.json]:
```suggestion
  "version": "0.2.1-alpha.1",
```

**Review by DenisHdz — APPROVED** [review]
Apart from the versioning, LGTM.

**Review by thekp — APPROVED** [review]

**Pull request merged**